### PR TITLE
Feature/cax portal@cplp 408 email registration next steps

### PR DESCRIPTION
**Pull request by Akshay-Chaudhari-BMW**
CPLP - 408 Email Registration Next Steps Dev.

### CODE REVIEW

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs [hunk 1]
```diff
@@ -153,9 +154,23 @@ public async Task CreateDocument(IFormFile document, string userName)
                     hash = builder.ToString();
                 }
             }
-            await _dbAccess.UploadDocument(name,documentContent,hash,userName);
+            await _dbAccess.UploadDocument(name, documentContent, hash, userName);
```
Comment (evegufy) [inline coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs]: Cleaning up is in general good but in general I wouldn't recommend to remove or add spaces in parts which don't have anything to do with your implementation, if you do that please do that in a separate commit, at least in the future

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "LogLevel": {
+          "Default": "Information",
+          "Microsoft": "Warning",
+          "Microsoft.Hosting.Lifetime": "Information"
+
```
Comment (evegufy) [inline coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json:7]: Please make sure not to commit any changes to the appsettings.Development.json

File: coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json [hunk 3]
```diff
@@ -6,4 +6,4 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   }
-}
+}
```
Comment (evegufy) [inline coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json:9]: Please make sure not to commit any changes to the appsettings.Development.json

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs [hunk 4]
```diff
@@ -153,9 +154,23 @@ public async Task CreateDocument(IFormFile document, string userName)
...
         }
         public Task CreateCustodianWalletAsync(WalletInformation information) =>
             _custodianService.CreateWallet(information.bpn, information.name);
+
+        public async Task<bool> SubmitRegistrationAsync(string userEmail)
+        {
+            var password = new Password().Next();
```
Comment (evegufy) [inline coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs]: Why is the password needed? I don't think that is need in the nextsteps template.

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json (hunk 2 above)
Comment (evegufy) [inline coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json]: Please make sure not to commit any changes to the appsettings.Development.json

File: coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json (hunk 3 above)
Comment (evegufy) [inline coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json]: Please make sure not to commit any changes to the appsettings.Development.json

**Review by evegufy — CHANGES_REQUESTED** [review]

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs (hunk 4 above)
Comment (Akshay-Chaudhari-BMW) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs]: password implementation removed

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json (hunk 2 above)
Comment (Akshay-Chaudhari-BMW) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json]: accidentally committed the changes and later removed.

File: coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json (hunk 3 above)
Comment (Akshay-Chaudhari-BMW) [reply coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json]: accidentally committed the changes and later removed.

**Review by evegufy — APPROVED** [review]

**Pull request merged**